backend/routes/api/users.js

```diff
@@ -2,9 +2,9 @@ const express = require("express");
 const router = express.Router();
 const User = require("../../models/User");
 const { celebrate, Joi } = require("celebrate");
-
+const auth = require("../../utils/auth-middleware");
 // get all users
-router.get("/", async (req, res) => {
+router.get("/", auth.isAuthenticated, async (req, res) => {
```

Comment:
This will be an `auth.isAdmin` endpoint since only the admins should be able to get all users. But #38 will fix this.